Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
     sox \
     swig \
     && apt-get clean && rm -rf /var/lib/apt/lists/* \
-    && pip3 --no-cache-dir install pycdio==2.1.0 discid
+    && pip3 --no-cache-dir install pycdio==2.1.0 discid pillow
```

Comment:
I think you can avoid using pip to install this module (using the apt dependency named `python3-pil`).